tests/run.go

```diff
@@ -620,9 +620,7 @@ func (t *test) run() {
 		os.Setenv("GOOS", runtime.GOOS)
 	}
 	if os.Getenv("GOARCH") == "" {
-		//os.Setenv("GOARCH", runtime.GOARCH)
-		// GOPHERJS.
-		os.Setenv("GOARCH", "js") // We're running this script natively, but the tests are executed with js architecture.
+		os.Setenv("GOARCH", runtime.GOARCH)
```

Comment:
The commit message of 62bca28b2835514b22712edb8104a624030a08c0 described this change as:

> Set "GOARCH" env var to correct value for gopherjs in tests/run.go.

Doing that is no longer needed, since GopherJS now ignores GOARCH. I'd rather revert it and keep tests/run.go more similar to upstream, rather than continuing to deviate it unnecessarily.